examples/ASP.NET Core 3/ASP.NET Core 3 - VS2019/NLog.config

```diff
@@ -13,20 +13,26 @@
 
   <!-- the targets to write to -->
   <targets>
-    <!-- write logs to file  -->
+    <!-- File Target with simple details for all log messages  -->
     <target xsi:type="File" name="allfile" fileName="c:\temp\nlog-AspNetCore3-all-${shortdate}.log"
             layout="${longdate}|${event-properties:item=EventId_Id:whenEmpty=0}|${uppercase:${level}}|${logger}|${message} ${exception:format=tostring}" />
 
-    <!-- another file log, only own logs. Uses some ASP.NET core renderers -->
+    <!-- File Target with more details using some ASP.NET core renderers -->
```

Comment:
While it's of course in the rules, but not sure if starters will get that, should we not mention that only the "own logs" are logged here?

Reply:
You are welcome to improve with more details, without it turning into a novel :)

Reply:
Have updated the comment with "own logs"